content/id/docs/reference/kubectl/cheatsheet.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Contekan kubectl
```

Comment:
Is contekan already included in kbbi?

Reply:
sontekan is the formal form https://kbbi.kemdikbud.go.id/entri/sontekan, but contekan is widely more understandable. wdyt we should use?
ccc @kubernetes/sig-docs-id-reviews